src/docfx.website.themes/common/common.js

```diff
@@ -155,9 +158,7 @@ function getGitContributeFallbackWithGitRemote(gitRemote, gitContribute) {
     }
 
     if (gitContribute) {
-        // apiSpecFolder defines the folder contains overwrite files for MRef
-        // the default value is apiSpec
-        modelGitInfo.apiSpecFolder = gitContribute.path || "apiSpec";
+        modelGitInfo.apiSpecFolder = gitContribute.path;
```

Comment:
different implementation same effect?

---
In reply to: [91675728](https://github.com/dotnet/docfx/pull/979#discussion_r91675728) [](ancestors = 91675728)